docs/contributing/Style-guide-for-Flutter-repo.md

```diff
@@ -30,9 +31,8 @@ understanding with new team members.
 The primary goal of these style guidelines is to improve code readability so
 that everyone, whether reading the code for the first time or
 maintaining it for years, can quickly determine what the code does.
-Secondary goals are to design systems that are simple, to increase the
-likelihood of catching bugs quickly, and avoiding arguments when there are
-disagreements over subjective matters.
+Secondary goals are to design systems that are simple and bug-free, 
```

Comment:
```suggestion
Secondary goals are to design systems that are simple and bug-free,
```

It looks like [**Linux Analyze**](https://ci.chromium.org/ui/p/flutter/builders/try/Linux%20analyze/81197/overview) failed because of the trailing space here 😃

Reply:
Thanks! Fixed.